src/test/java/org/cactoos/map/MergedTest.java

```diff
@@ -35,6 +37,27 @@
  */
 public final class MergedTest {
 
+    @Test
+    public void behavesAsMapCreatedFromIterable() {
+        new Assertion<>(
+            "Must behave as a map",
+            new Merged<>(
+                new MapOf<>(
+                    new MapEntry<>("a", 1)
+                ),
+                new MapOf<>(
+                    new MapEntry<>("b", 2)
+                )
+            ),
```

Comment:
@baudoliver7 Actually not, there's not need to test  `Iterable` ctor because it is primary and getting hit anyway.

Reply:
@andreoss That's right ! But it's because of the test name (`behavesAsMapCreatedFromIterable`) that I said that. However, you change it in your last commit.